CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Zowe CLI package will be documented in this file.
 
 ## Recent Changes
 
+- Added new method "dataSetLike(session, dataSetName, options)" to `Create` class, for use when creating a dataset with parameters like another data set
 - Fix compilation error
```

Comment:
Please merge master into your branch and resolve conflicts. The line "Fix compilation error" is no longer needed here since it was fixed in 6.19.1.